package/com.unity.formats.usd/Tests/Editor/ImportMeshTests.cs

```diff
@@ -0,0 +1,146 @@
+using System.IO;
+using NUnit.Framework;
+using pxr;
+using UnityEditor;
+using UnityEngine;
+using USD.NET;
+
+namespace Unity.Formats.USD.Tests
+{
+    public class ImportSkinnedMeshTests : BaseFixtureEditor
+    {
+        static string k_testAssetGUID = "3d00d71254d14bdda401019eb84373ce";
+        GameObject m_usdRoot;
+
+        [SetUp]
+        public void SetUp()
+        {
+            InitUsd.Initialize();
```

Comment:
I think since BaseFixture has a setup step that includes InitUsds.Initialize(), this one might be redundant

Reply:
Good point. I'll remove it.